### ISSUE TITLE
Wiki for System Description

### ISSUE THREAD

**Issue by FadySalama** [tum-esi/wot-system-description#8]
There is a need to write a wiki or a specification-style document that describes the rules and syntax of the SD in both UML and JSON-LD format

**Comment by egekorkan** [comment]
We can do it as a wiki here but I think that a w3c style spec would look much nicer and convincing ;). We can then host it here as a GitHub page :)

**Comment by egekorkan** [comment]
Yes of course :) I will be using respec: https://github.com/w3c/respec
